Assets/HoloToolkit-Examples/SharingWithUNET/Scripts/NetworkDiscoveryWithAnchors.cs

```diff
@@ -94,24 +95,37 @@ private void MaybeInitAsServer()
                 return;
             }
 
+            StartCoroutine(InitAsServer());
+        }
+
+        private IEnumerator InitAsServer()
+        {
             Debug.Log("Acting as host");
             // StopBroadcast will also 'StopListening'
             StopBroadcast();
 
+            // Work-around when building to the HoloLens with "Compile with .NET Native tool chain".
+            // Need a frame of delay after StopBroadcast() otherwise clients won't connect.
+            yield return null;
```

Comment:
Do you know what the underlying issue is that requires waiting a frame?  What makes one frame enough time to wait?  It would be ideal if there's some other state that could be checked to see if the system is ready to successfully perform the next step.